go.mod1

```diff
@@ -14,3 +14,5 @@ require (
 	github.com/sirupsen/logrus v1.2.0
 	golang.org/x/text v0.3.0
 )
+
+replace github.com/pingcap/tidb => github.com/feloxx/tidb v0.0.0-00000000000000-4df31ec070b4c0971059a02ed576802a334378b8
```

Comment:
Please file the corresponding PR on TiDB for this.

cc @tiancaiamao This PR causes a "cyclic dependency" between TiDB and Parser, both need to be updated together. What is our conventional sequence to make it work while keeping the CI passing? (or is it too much a hassle which the whole PR should be rejected?)